### DISCUSSION TITLE
Where's CSS!!!

### DISCUSSION THREAD

**Discussion by rayhanadev** [whippingdot/Language-Tutorials#17] [Add Page]
Sounds like a job for me :)
Maybe I'll drop a tutorial

**Comment by whippingdot** [comment]
YAY GODLY RAYHAN !!!!

BTW create a discussion next time. Also do you want to join the org or contribute like an outsider.

**Comment by ghost** [comment]
This should be blocked on a discussion for adding XML or HTML, as they are the only languages that I am aware of that can be styled with CSS.

Also, the README.md gave a somewhat discouraging tone towards non-programming languages:
> Markdown doesn't count as a coding language but we are teaching it too.

**Comment by rayhanadev** [comment]
CSS actually can be a programming language but that's completely irrelevant to this conversation xD
And also I can contribute from outside I don't feel like join any organizations just yet.

@CrimsonCodes0 you're not wrong but I feel CSS can be learned on a separate tutorial because of how vast it actually is.

**Reply by whippingdot** [reply]
umm this isn't a organization. This is a project I made on my own lul. If you join, you will join the repo. That is all

**Reply by ghost** [reply]
@whippingdot You referred to it as an organization first,
> Also do you want to join the org...

@RayhanADev A yes, I should've clarified: I was referring to XML/HTML. More often than not, I tend to defend the "CSS is a programming language" stance myself.

But how are you going to explain CSS without actually styling something?
You can't really explain selectors without a corresponding DOM structure, can you?

Are you going to write,
```css
foo {
    color: blue;
}
```
without ever showing an example of what `foo` actually refers to?

( Of course, if there were a standalone use for CSS, my suggestions would be incorrect ;) )

**Reply by rayhanadev** [reply]
Ahhh you're not wrong @CrimsonCodes0 , not wrong at all...

hmm, I guess a tie-in maybe? explain some basic HTML, then redirect the attention to a CSS tutorial, explain CSS, and then back to advanced HTML topics?

**Reply by ghost** [reply]
@RayhanADev I'm not sure what approach you want to go with, but most tutorials I've encountered about CSS or HTML often include bits of the other; it's hard to find a standalone HTML or CSS tutorial, with the exception of JS<->HTML tutorials that don't focus on the CSS.

**Comment by whippingdot** [comment]
Hey @RayhanADev get on this when ur free alright?

**Reply by rayhanadev** [reply]
kk

**Reply by whippingdot** [reply]
Thanks!
